hybrid/decomposers.py

```diff
@@ -930,7 +1066,7 @@ def make_origin_embeddings(qpu_sampler=None, lattice_type=None,
         lattice-structured problems <https://arxiv.org/abs/2202.03044>`_
     """
     if qpu_sampler is None:
-        if lattice_type == 'pegasus' or lattice_type == 'chimera':
+        if lattice_type is not None:
```

Comment:
QPU doesn't support topologies like `kings` or `cubic`, so we should do an explicit check here, e.g:
```suggestion
        if lattice_type is in ['chimera', 'pegasus', 'zephyr']:
```
(assuming you want to add support for Zephyr, see line 1113, otherwise without `'zephyr'`)